### PR TITLE
Update more-about-chmod.md

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: linux/user-and-file-management/file-permissions/more-about-chmod.md [hunk 1]
```diff
@@ -26,58 +28,67 @@ Different digits in `chmod` mean a different set of permissions. The digits we c
...
-In the binary format they would look like this:
 ```sh
-777 = 111 111 111
 700 = 111 000 000
 ```
 
-> 💡 `777` means the owner, the groups, and other can read, write and execute permissions. While the `700` means only the owner has all three permissions while the groups and other have none.
-
-For general files:
+Now we can reference the table above and see that `111 000 000` means that the owner has read, write and execute permissions while the groups and other have none.
```
Comment (nem035) [inline linux/user-and-file-management/file-permissions/more-about-chmod.md:47]:
```suggestion
Now we can reference the table above and see that `111 000 000` means that the owner has read, write and execute permissions while the groups and other users have none.
```

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Review by Stefan-Stojanovic — APPROVED** [review]
Looks good

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/linux/user-and-file-management/file-permissions/more-about-chmod.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/linux/user-and-file-management/file-permissions/more-about-chmod.md?ref=f60e83beefc770b710f877cb2302c3e3b5d6b35f)

**Pull request merged**